scraps/get_user_input.py

```diff
@@ -0,0 +1,28 @@
+import tkinter as tk
+from tkinter import filedialog, simpledialog
+
+root = tk.Tk()
+root.withdraw() # Hide the main window
+root.call('wm', 'attributes', '.', '-topmost', True)
+
+xlsx_path = filedialog.askopenfilename(title="Open Excel-bestand met titel en artiesten...", filetypes=[("Excel Files", "*.xlsx")])
+print("Locatie Excelbestand:", xlsx_path)

```

Comment:
als ik in m'n IDE een path doorgeef zoals: "image_path="C:\Users\marti\Pictures\Sram 4.jpg"" 
zorgt het woord "user"  voor een probleem. 
![image](https://github.com/MartijnvCitteren/music_bingo/assets/141489106/86cffcaf-87c1-468f-b893-cad02f3cf684)